docs/self-managed/setup/guides/add-extra-manifests.md

```diff
@@ -0,0 +1,39 @@
+---
+id: add-extra-manifests
+title: "Add extra manifests to the Camunda 8 Helm chart"
+sidebar_label: "Add extra manifests"
+description: "Learn how to add extra manifests to Helm deployments by injecting arbitrary data in the values.yaml."
+---
+
+## Overview
+
+When using the Camunda 8 [Helm chart](/self-managed/setup/install.md), arbitrary data can be injected into the `values.yaml` file, enabling the addition of extra Kubernetes manifests. This feature is particularly useful for adding custom manifests by including additional Kubernetes resources such as ConfigMaps, Deployments, or Services.
+
+## Usage
+
+Extra manifests are defined within your `values.yaml` file using the following syntax:
+
+```yaml
+global:
+  extraManifests:
+    - |
+      apiVersion: v1
+      kind: ConfigMap
+      metadata:
+        name: example-cm
+      data:
+        test: test
+```
+
+For more information, see the Kubernetes [object documentation](https://kubernetes.io/docs/concepts/overview/working-with-objects/).
```

Comment:
I wanted to provide a little more information here. Is this the best Kubernetes link? 

I was also wondering if we could give any additional context, like:
- Can you add more than one manifest? What is the format for adding more than one? 
- Does the chart automatically create these resources when deployed? 
- What does maintenance look like for updating these objects? (...This might just be my ignorance here.)

Based on https://github.com/camunda/product-hub/issues/2464, it seems like the use case is deployment customization that doesn't involve modifying the Helm chart - I can call that out specifically, too, with any of the answers to the above.

Reply:
yes, you can add more manfiests. I can make that clearer by adding a second configmap in the array

> Does the chart automatically create these resources when deployed?
Helm will render the manifest and deploy the rendered manifests to the cluster. I don't think this point is worth mentioning int he guide.

> What does maintenance look like for updating these objects?
The customers maintains the extra manfiests themselves. That's why I listed the points in the best practices section. Also, I don't think this is worth adding to the guide.